.travis.yml

```diff
@@ -41,20 +41,20 @@ branches:
 matrix:
   include:
     - <<: *xcode
-      name: Xcode 10.2 / Swift 5.0
-      osx_image: xcode10.2
-      env: JOB=Xcode10.2
```

Comment:
Does this PR drop support for Swift 5.0 and/or Xcode 10.2's sdk? If so, that's probably something we should be cautious about.

Reply:
Yes, I believe it would drop support for Swift 5.0 because Quick 3.0.0 and Nimble 9.0.0 require Swift 5.2 and Xcode 11.4.

Alternatively, we could urge the maintainers of Quick and Nimble to port Apple Silicon support back to Quick 2.x and Nimble 8.x.

Incidentally, I think we'll still need to wait for a subsequent release of Quick, as https://github.com/Quick/Quick/commit/1172a5df7ee3da5817cf7c9771c10aeb93b770f4 wasn't in the 3.0.0 release.

Reply:
Got it, thanks!

Reply:
@ikesyo I would love to hear your quick thoughts about these Swift version changes as you've been handling that well for several of the mentioned projects.